index.html

```diff
@@ -48,6 +48,7 @@ <h1 id="logo"><a href="index.html">GreatUniHack</a></h1>
                     <ul>
                         <li><a href="index.html">Home</a></li>
                         <li><a href="#faq" class="scrolly">FAQ</a></li>
+                        <li><a href="#schedule" class="scrolly">Schedule</a></li>
                         <li><a href="#sponsors" class="scrolly">Sponsors</a></li>

```

Comment:
Consider having the sponsors section first and then the schedule one.

Reply:
Should I swap the actual sections around as well?

Reply:
Yes